README.md

```diff
@@ -245,6 +245,13 @@ Run the following command from the project root to create a new dashboard admin:
 bundle exec rails 'create_admin_user[<name>,<email>,<password>]'
 ```
 
+### View Sandbox data in your local
+1. Follow the steps in the "How to add an SSH key..." section [here](https://github.com/simpledotorg/deployment) to add your SSH key to the deployment repo
+2. Ask someone from the Simple team to add you as an admin to sandbox
+3. Create a password for your Sandbox account and use that to log into the Dashboard on localhost:3000
```

Comment:
You won't be able to do this on `localhost:3000` until after step 5. Should this `on https://api-sandbox.simple.org` instead?